src/main/java/cardibuddy/ui/FlashcardCard.java

```diff
@@ -0,0 +1,66 @@
+package cardibuddy.ui;
+
+import cardibuddy.model.flashcard.Flashcard;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+
+/**
+ * An UI component that displays information of a {@code deck}.
```

Comment:
should be flashcard

Reply:
> Seems okay and can be merged. Have you tested this out?

nope have not tested out! cos i need a command to open the deck but we haven't done it yet. will probably try to replace all the deckCard with flashcardcard just to see if it works